### PR TITLE
Add license year workflow

### PR DESCRIPTION
**Pull request by chartgerink**
This PR adds a workflow to automatically open PRs to update the license year. This helps prevent these from becoming outdated.

Currently it would run at 03.00AM on January 1st ([`0 3 1 1 *`](https://crontab.guru/#0_3_1_1_*)), which we can adjust if we want. The current text would read:

***
   This PR updates the copyright license for this new year! If you're reading this while you're celebrating, enjoy! Don't worry about this one :blush:
           Happy new year!
***

I am not 100% sure this will run as expected as I don't have everything installed to test this locally beforehand. It is an extension of what we use @libscie, adjusted to handle both LICENSE files present in the template.

### CODE REVIEW

File: .github/workflows/update-copyright-year.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+          prTitle: Update the license year
+          prBody: >
+           This PR updates the copyright license for this new year! If you're reading this while you're celebrating, enjoy! Don't worry about this one :blush:
+           ![Happy new year!](https://media.giphy.com/media/HyDfNCZlTn5iU/giphy.gif?cid=ecf05e4777yl7dbo1xfha6bx1z5lrl13uq7biv6rs9dqsyoh&ep=v1_gifs_search&rid=giphy.gif&ct=g)
+      - uses: FantasticFiasco/action-update-license-year@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: | 
+            LICENSE.md
+            LICENSE
```
Comment (Bisaloo) [inline .github/workflows/update-copyright-year.yml:24]: I **think** these arguments are in the wrong place:

```suggestion
      - uses: FantasticFiasco/action-update-license-year@v2
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          path: | 
            LICENSE.md
            LICENSE
          prTitle: Update the license year
          prBody: >
           This PR updates the copyright license for this new year! If you're reading this while you're celebrating, enjoy! Don't worry about this one :blush:
           ![Happy new year!](https://media.giphy.com/media/HyDfNCZlTn5iU/giphy.gif?cid=ecf05e4777yl7dbo1xfha6bx1z5lrl13uq7biv6rs9dqsyoh&ep=v1_gifs_search&rid=giphy.gif&ct=g)

```

**Review by Bisaloo — COMMENTED** [review]
Thanks! This relates to #101.

**Comment by chartgerink** [comment]
Thanks @Bisaloo for the catch - removed the `prTitle` given that it defaults to the same.

**Review by Bisaloo — APPROVED** [review]

**Comment by Bisaloo** [comment]
Tested here and it looks like it doesn't work for `LICENSE` out of the box: https://github.com/Bisaloo/testpkg/actions/runs/7957594497/job/21720690471

**Comment by chartgerink** [comment]
@Bisaloo I fixed the issue and tested it - see the successful test run here: https://github.com/chartgerink/testpkg/pull/2/files

Happy to hear your renewed thoughts 😊

**Review by Bisaloo — APPROVED** [review]
Thanks! I was initially unsure about the year range rather than a single year since the latter is more common in the R community but it seems that both are valid.

**Pull request merged**